tests/foreman/cli/test_satellitesync.py

```diff
@@ -16,95 +16,121 @@
 from robottelo.cli.repository import Repository
 from robottelo.cli.repository_set import RepositorySet
 from robottelo.cli.settings import Settings
 from robottelo.cli.subscription import Subscription
-from robottelo.config import settings
+from robottelo.constants import CONTAINER_REGISTRY_HUB
 from robottelo.constants import DEFAULT_CV
-from robottelo.constants import ENVIRONMENT
 from robottelo.constants import PRDS
 from robottelo.constants import REPOS
 from robottelo.constants import REPOSET
 
 
 @pytest.fixture(scope='class')
 def class_export_directory():
-    """Create a directory for export, configure permissions and satellite settings,
-    at the end remove the export directory with all exported repository archives"""
-    export_dir = gen_string('alphanumeric')
-
-    # Create a new 'export' directory on the Satellite system
-    result = ssh.command(f'mkdir /mnt/{export_dir}')
-    assert result.return_code == 0
-
-    result = ssh.command(f'chown foreman.foreman /mnt/{export_dir}')
-    assert result.return_code == 0
-
-    result = ssh.command(f'ls -Z /mnt/ | grep {export_dir}')
-    assert result.return_code == 0
-    assert len(result.stdout) >= 1
-    assert 'unconfined_u:object_r:mnt_t:s0' in result.stdout[0]
-
-    # Fix SELinux policy for new directory
-    result = ssh.command(f'semanage fcontext -a -t foreman_var_run_t "/mnt/{export_dir}(/.*)?"')
-    assert result.return_code == 0
-
-    result = ssh.command(f'restorecon -Rv /mnt/{export_dir}')
-    assert result.return_code == 0
-
-    # Assert that we have the correct policy
-    result = ssh.command(f'ls -Z /mnt/ | grep {export_dir}')
-    assert result.return_code == 0
-    assert len(result.stdout) >= 1
-    assert 'unconfined_u:object_r:foreman_var_run_t:s0' in result.stdout[0]
-
-    # Update the 'pulp_export_destination' settings to new directory
-    Settings.set({'name': 'pulp_export_destination', 'value': f'/mnt/{export_dir}'})
-    # Create an organization to reuse in tests
-    yield export_dir
-    ssh.command(f'rm -rf /mnt/{export_dir}')
+    """Grab export directory at the end remove the export directory with
+    all exported repository archives"""
 
 
-class ExportDirectoryNotSet(Exception):
-    """Raise when export Directory is not set or found"""
+@pytest.fixture(scope='class')
+def config_export_import_settings():
+    """Check settings and set download policy for export.  Reset to original state after import"""
+    download_policy_value = Settings.info({'name': 'default_download_policy'})['value']
+    rh_download_policy_value = Settings.info({'name': 'default_redhat_download_policy'})['value']
+    disconnected_mode_value = Settings.info({'name': 'content_disconnected'})['value']
+    assert download_policy_value == 'immediate'
+    assert rh_download_policy_value == 'on_demand'
+    assert disconnected_mode_value == 'false'
+    Settings.set({'name': 'default_download_policy', 'value': 'immediate'})
```

Comment:
Why do you call `set` on this when it's asserted to be `immediate` already on L60?

Reply:
It's suppose to be `rh_download_policy_value` that gets set, not `download_policy_value`.